x-pack/test/reporting_api_integration/services/scenarios.ts

```diff
@@ -197,7 +198,7 @@ export function createScenarios({ getService }: Pick<FtrProviderContext, 'getSer
     const {
       body: [job],
     } = await supertestWithoutAuth
-      .get(`${INTERNAL_ROUTES.JOBS.LIST}?page=0&ids=${id}`)
+      .get(`/api/reporting/jobs/list?page=0&ids=${id}`)
```

Comment:
same as https://github.com/elastic/kibana/pull/162517/files#r1280991975

Reply:
whomp :face_palm: thank you